### ISSUE TITLE
How to compare available fixers and the ones I actually use

### ISSUE THREAD

**Issue by Aerendir** [PHP-CS-Fixer/PHP-CS-Fixer#3045]
PHPCS Fixer provides a lot of fixers.
The configuration can be also nested and customized.
New fixes also are added constantly.

So it becomes really hard to understand which fixers are actually really used and which ones, instead, aren't.

But there are some fixers that can be really useful.

Unfortunately discover those fixers is really hard as someone has to go fixer by fixer to check if it is used by some of the used groups or if it set in in the custom `.php_cs`.

So a command that provides a list of all available fixers comparing them with the ones actually used should be really useful, doesn't it?

**Comment by SpacePossum** [comment]
I agree, I think something like this was proposed here as well; https://github.com/FriendsOfPHP/PHP-CS-Fixer/issues/2996

**Comment by Aerendir** [comment]
Keep open

**Comment by Aerendir** [comment]
keep
